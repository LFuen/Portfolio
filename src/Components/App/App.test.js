import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { render, screen } from '@testing-library/react';
import App from './App';
import { BrowserRouter } from 'react-router-dom';

describe(`About component`, () => {
  describe(`Snapshot`, () => {
    it(`renders the UI as expected`, () => {
      const app = renderer.create(
        <BrowserRouter>
          <App />
        </BrowserRouter>).toJSON()
      expect(app).toMatchSnapshot()
    })

  })

  describe(`Smoke test`, () => {
    it('renders without crashing', () => {
      const div = document.createElement('div')

      ReactDOM.render(
        <BrowserRouter>
          <App />
        </BrowserRouter>, 
        div
      )
      ReactDOM.unmountComponentAtNode(div)
    })    
  })
})