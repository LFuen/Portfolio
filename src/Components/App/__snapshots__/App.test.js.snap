// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component Snapshot renders the UI as expected 1`] = `
<div>
  <nav
    className="navigation"
  >
    <a
      aria-current={null}
      className="link"
      href="/Portfolio"
      onClick={[Function]}
    >
      Welcome
    </a>
    <a
      aria-current={null}
      className="link"
      href="/About"
      onClick={[Function]}
    >
      About
    </a>
    <a
      aria-current={null}
      className="link"
      href="/Projects"
      onClick={[Function]}
    >
      Projects
    </a>
    <a
      aria-current={null}
      className="link"
      href="/Contact"
      onClick={[Function]}
    >
      Contact
    </a>
    <a
      aria-current={null}
      className="link"
      href="/Inquiries"
      onClick={[Function]}
    >
      Inquiries
    </a>
  </nav>
  <footer
    className="foot"
  >
    <p>
       Â© Lili Fuentes | 2020 
    </p>
  </footer>
</div>
`;
